.github/workflows/README.md

```diff
@@ -0,0 +1,89 @@
+| publish-to-test   |    ❌     |                 true                  | boolean | If set to true, the packages are published to test.pypi.org other wise the packages are published to pypi.org                      |
+| python-version    |    ❌     |                "3.10"                 | string  | The python version for setting up poetry.                                                                                          |
+| poetry-version    |    ❌     |               "1.1.12"                | string  | The poetry version to be installed.                                                                                                |
+| working-directory |    ❌     |                 "./"                  | string  | The working directory of your Python package.                                                                                      |
+
+## Secret Parameters
+
+These secrets define the GitHub user that pushes the changes of your `pyproject.toml` file to the repository. Create a
+repository secret for the GitHub username (`GH_USERNAME`), the GitHub Email (`GH_EMAIL`), and a personal access
+token (`GH_TOKEN`) of the user. You can use the no reply GitHub email for the
+email: `[username]@users.noreply.github.com`.
+
+| Name            |  Required  | Description                                    |
+|-----------------|:----------:|------------------------------------------------|
+| github-username |     ✅      | The GitHub username for pushing                |
+| github-email    |     ✅      | The GitHub email for pushing                   |
+| github-token    |     ✅      | The GitHub token for pushing                   |
+| pypi-token      |     ✅      | The (test) pypi api token for pushing packages |
+
+## Outputs
+
+This workflow outputs two variables: The `old-tag` and the `release-tag`. These variables can be used in the future
+jobs (e.g., using the `release-tag` to create GitHub release).
+
+| Name        | Description                                         |
+|-------------|-----------------------------------------------------|
+| old-tag     | Defines the old version in your pyproject.toml file |
+| release-tag | The bumped version of your project                  |
+
+## Calling the workflow
+
+```yaml
+name: Call this reusable workflow
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  call-workflow-passing-data:
+    uses: bakdata/ci-template/python-poetry-release.yaml@main
+    with:
+      release-type: patch # (Required) See more values at: https://python-poetry.org/docs/cli/#version
+      ref: my-awesome-ref # (Optional) if not set the ${{ github.event.repository.default_branch }} will fill the value. In this case the changes will be pushed to my-awesome-ref
+      publish-to-test: false # (Optional) Default value true. In this case the packages are pushed to PyPI
+      python-version: 3.8 # (Optional) Default value is 3.10. In this case poetry is installed with Python 3.8
```

Comment:
please change it here as well

Reply:
missed one 😛 this still shows 3.8 for me

Reply:
@disrupted This is just an example showing people that is possible to use Python `3.8`. Changing this to 3.10 does not really add a value to the example since 3.10 is already the default value.

Reply:
Also, the comment is describing clearly why the 3.8 us chosen :smile:

Reply:
ah ok, understood!